__tests__/submissions.test.ts

```diff
@@ -248,14 +273,309 @@ describe('getSubmissionById', () => {
             }
         }
 
-        const submission = await request(url).post('/').send(submissionQuery)
+        // Get error data
+        const searchData = await request(url).post('/').send(submissionQuery)
 
         // Compare the data returned in the response to the updated fields that were sent
-        const submissionErrorResponse = submission.body
+        const submissionErrorResponse = searchData .body
```

Comment:
You seem to have an empty space between each of the `searchData .` 
```suggestion
        const submissionErrorResponse = searchData.body
```